SayTheirNames/Source/View/Home/Person/PersonCell.swift

```diff
@@ -33,7 +33,7 @@ final class PersonCell: UICollectionViewCell {
     private lazy var dateOfIncidentLabel: UILabel = {
         let lbl = UILabel()
         lbl.text = "25.05.2020"
-        lbl.font = UIFont(name: "Karla-regular", size: 13)
+//        lbl.font = UIFont(name: "Karla-regular", size: 13)
```

Comment:
```suggestion
```

Reply:
sorry for the dead code. fixed

Reply:
please see my response to hybridcatttre: trait changes